### PR TITLE
[Feature1]First Feature was implemented

### PR DESCRIPTION
**Pull request by azizehahmadi**
Following changes were made:
- new-feature.txt was created 
- feature 1 folder was created

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+this file also added in feature1 branch
```
Comment (amir20hamadi) [inline feature1/one-more-file.txt:1]: its really good to add this line

Comment (amir20hamadi) [inline feature1/one-more-file.txt:1]: its really good to add this line

Comment (azizehahmadi) [reply feature1/one-more-file.txt:1]: thankyou

**Review by amir20hamadi — APPROVED** [review]
good job azizeh

**Pull request merged**